inc/audiences/namespace.php

```diff
@@ -82,6 +77,23 @@ function register_post_type() {
 	);
 }
 
+/**
+ * Force audiences to be sorted according to menu order by default.
+ *
+ * @param WP_Query $query The current query object.
+ */
+function pre_get_posts( WP_Query $query ) {
+	// Must be an exact query for the audiences post type.
+	if ( $query->get( 'post_type' ) !== POST_TYPE ) {
+		return;
+	}
+
+	$query->set( 'orderby', [
+		'menu_order' => 'ASC',
+		'post_title' => 'ASC',
+	] );
```

Comment:
Should this be a default?

Reply:
Yep, 100%

Reply:
Misunderstood - now respects existing orderby params.